relayer/src/chain/cosmos.rs

```diff
@@ -546,9 +613,13 @@ impl CosmosSdkChain {
         Ok(self.account()?.sequence)
     }
 
-    fn incr_account_sequence(&mut self) -> Result<(), Error> {
-        self.account()?.sequence += 1;
-        Ok(())
+    fn incr_account_sequence(&mut self) {
+        match &mut self.account {
+            Some(account) => {
+                account.sequence += 1;
+            }
+            None => {}
+        }
```

Comment:
I think this could be written slightly more simply as:

```suggestion
        if let Some(account) = &mut self.account {
                account.sequence += 1;
        }
```